src/app/shared/components/notification-card/notification-card.component.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { NotificationCardComponent } from './notification-card.component'
+import { instance, mock } from '@typestrong/ts-mockito'
+import { ElementRef } from '@angular/core'
+
+describe('NotificationComponent', () => {
```

Comment:
It says `NotificationComponent` but shouldn't it say `NotificationCardComponent`?

Reply:
Yes it should. Webstorm doesn't refactor magic strings when changing the name of a component.
It's been updated.